DataManagementSystem/Client/FTSFile.py

```diff
@@ -76,7 +76,7 @@ def tableDesc():
                "FileID": "INTEGER NOT NULL",
                "OperationID": "INTEGER NOT NULL",
                "RequestID": "INTEGER NOT NULL",
-               "LFN": "VARCHAR(255) NOT NULL",
+               "LFN": "VARCHAR(955) NOT NULL",
```

Comment:
Well, 1024 is anyway a very large number, and I would definitely stick to power of 2, so I would be in favor of having both at 1024. If someone feels the need for more (Which I would find surprising anyway), they can always increase in their DB